src/mapcss/Eval.py

```diff
@@ -30,6 +30,21 @@ def __init__(self, s='eval()'):
         except:
             # print "Can't compile %s" % s
             self.expr = compile("0", "MapCSS expression", "eval")
+    
+    def __eq__(self, a):
+        return self.expr_text == a.expr_text
+    
+    def to_mapbox_expression(self):
+        def to_mapbox_coalesce(*x):
+            return ["coalesce"] + list(x)
```

Comment:
todo: handle coalesce on db side

Reply:
mmm no.

`any` in style is a part of the style and should be on the rendering side - it should work if you change the style.

db can handle transparent augmentations like "add more english names" that could have been, but were not, in the data.

Reply:
agree